en/docs/guides/access-delegation/openid-connect-discovery.md

```diff
@@ -0,0 +1,227 @@
+## Discover OpenID Connect Provider
+
+This page guides you through using [OpenID Connect Discovery](insertlink) to discover an end user's OpenID provider, and to obtain information required to interact with the OpenID provider, including its OAuth 2.0 endpoint locations. 
+
+The OpenID Connect discovery endpoint is as follows:
+
+```
+https://localhost:9443/.well-known/oidcdiscovery
+```
+
+-----
+
+## OpenID Provider Issuer discovery
+
+OpenID Provider Issuer discovery refers to the process of determining the location of the OpenID Provider.
+
+In WSO2 Identity Server, the default OpenID Provider Issuer location path is set to `oidcdiscovery/.well-known/openid-configuration` .
+To move the OpenID provider issuer location path to the root `<issuer>/.well-known/openid-configuration`, add the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.
+    
+``` java
+[oauth.endpoints]
+oidc_discovery_url= "${carbon.protocol}://${carbon.host}:${carbon.management.port}/oauth2/token</"
+```
+
+----
+
+## Configure the OpenID Provider issuer location
+
+In WSO2 Identity Server, the resident IdP Entity ID for OpenID Connect can be configured as the OpenID Provider Issuer location. 
+
+1.  Add the following property to the `deployment.toml` file found in the `<IS_HOME>/repository/conf/` folder.
+
+    ``` java
+    [oauth]
+    use_entityid_as_issuer_in_oidc_discovery= "true"
+    ```
+
+    !!! warning
+        In future releases, the Entity ID will be used as the OpenID
+        Provider Issuer location by default and will not need to be enabled
+        manually using the property mentioned above. Therefore, the
+        `            use_entityid_as_issuer_in_oidc_discovery           `
+        property will be deprecated in the next release.
+    
+
+2.  Log in to the management console.
```

Comment:
Aren't we replacing the management console steps yet?

Reply:
since we don't have content for the admin portal yet, I added the console steps for now